components/BookingOverview/index.js

```diff
@@ -16,16 +19,28 @@ export default function BookingOverview({
   return (
     <>
       <CardStyled>
-        <ImageStyled
-          src="/cowgirl_on_white_horse.jpg"
-          alt="cowgirl on white horse"
-          height={220}
-          width={332}
-        />
-        <p>Datum: {bookedTimeslot?.currentDate}</p>
-        <p>Uhrzeit: {bookedTimeslot?.startTime}</p>
-        <div>Personen: {recentlyBooked?.numberOfPeople}</div>
-        <p> Pferde:</p>
+        <DeleteButtonOverview
+          type="button"
+          onClick={() =>
+            onHandleDelete(
+              recentlyBooked.timeslot_id,
+              recentlyBooked.booking_id
+            )
+          }
+        >
+          x
+        </DeleteButtonOverview>
+        <DateStyled>
+          <BoldPTag>Datum:</BoldPTag> {bookedTimeslot?.currentDate}
+        </DateStyled>
+        <p>
+          <BoldPTag>Uhrzeit: </BoldPTag>
+          {bookedTimeslot?.startTime}
+        </p>
```

Comment:
A paragraph inside another paragraph could be confusing for semantic reasons. Maybe make the outer paragrapth a div instead?